### DISCUSSION TITLE
Is Stellarium directly developed according to any astrological system?

### DISCUSSION THREAD

**Discussion by pheom** [Stellarium/stellarium#1780] [General]
Hi, I just created this account in order to know the answer of the title of this post.

I am not trying to create any philosophical, astrological or any deep discussion. I am just a casual student and follower of astrology, and recently has come to my awareness that the skies are "changing". At the same time I was referred to Stellarium where I could see for myself. I have no ties to astronomy though, which is probably a mistake.

Right now in 'western astrology', if you check any 'chart of the moment' they depict that, for example, Venus and Mars are in Leo and the Sun recently crossed over to Cancer. Using Stellarium, it shows a different picture. Venus/Mars in Cancer and Sun in Gemini. These are just some examples, as I can see, every single point that I checked went one sign backwards (even Pluto, apparently, that is so slow moving).

However, Stellarium seems to be coherent with the Hindu Vedic system, which to my knowledge was always somewhat different from the Western one. 

Hence the question in the title of this post. This might be huge, it might be an actual change, or it might mean that the 'western' astrologers have always been dellusional or simply liars.

I am not familiar with the environment of this forum, I am just honestly trying to know. Please, let's refrain from debating if astrology is or not of any meaning or any discussion that might instigate hate.

All the best to all

**Comment by alex-w** [comment]
Stellarium is astronomical software, not an astrological!

**Comment by gzotti** [comment]
Stellarium and its developers simply do not care about any astrological system, so we cannot answer this question. Stellarium is astronomy software based on 20th/21st century definitions, textbooks and algorithms. Some astrologies use compatible algorithms to compute their astronomy, others don't. Followers of astrology must find out compatibilities themselves. 

This is like the question towards authors of some chemistry software, whether it is usable for alchemic studies from 16th century textbooks. How should the developers know? They likely studied the state of modern chemistry in the past 20 years.

**Reply by pheom** [reply]
Fair point. Thank you for the answer :)

**Reply by axd1967** [reply]
From what I understood, Hindu astrology is based on a sidereal zodiac and takes precession into account (see https://en.wikipedia.org/wiki/Hindu_astrology#History_and_core_principles), while Western astrology uses the **tropical** zodiac, which is aligned to the equinoctial points (https://en.wikipedia.org/wiki/Astrology#Western).  See also https://en.wikipedia.org/wiki/Sidereal_and_tropical_astrology

**Reply by LioraFarkovitz** [reply]
I want to be able to see it in Stellarium and also look on the world wide telescope. Has anyone else been watching 'Electric Universe' on Gaia? That is a real game changer on every level, if it's true. The hypothesis is that our solar system functions on electrical principles. That would marry astronomy and astrology in an interesting hypothesis. 

I'm interested in solar power for sustainable housing, and I like the smartflower solar system. There's some math out here somewhere for it to follow the sun during the day. But I wonder, what keeps the system going all night following and documenting the stars, or triangulating with a space telescope?

**Reply by gzotti** [reply]
> if it's true

That's the question which we don't discuss here. Stellarium illustrates "traditional academic knowledge".

You don't need a space telescope to drive a solar panel. 2nd century math is enough. Implemented on 20th-century computers, motors and batteries. Nothing revolutionary. But still not enough of them.

**Reply by LioraFarkovitz** [reply]
After the last few years, there really isn't a standard for "traditional academic knowledge", thanks to Jeffrey Epstein & Company, Anthony Fauci, and Bill Gates. The math is the same, what does it hurt? It's fun and it's a good programming challenge for students. So, I say "Ladidadidadidah" to your "traditional academic knowledge".

Also, it seems like something like a Smartflower solar system might take interesting photographs for astronomers and astrophysicists. They may not be as powerful as the ones in universities today, but they still have to be more powerful than what those institutions used a short time ago.

**Comment by AMAkidsCyprus** [comment]
yeah
